app/src/components/widgets/charts/launchExecutionAndIssueStatistics/launchExecutionAndIssueStatisticsTooltip.scss

```diff
@@ -0,0 +1,38 @@
+.item-wrapper {
+  margin: 3px 0;
+}
+
+.color-mark {
+  position: relative;
+  top: 1px;
+  display: inline-block;
+  width: 12px;
+  height: 12px;
+  margin-right: 4px;
+  border-radius: 6px;
+}
+
+.launch-name {
+  width: 100%;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  font-family: $FONT-SEMIBOLD, sans-serif;
+}
+
+.launch-start-time {
+  margin-bottom: 4px;
+}
+
+.item-name {
+  margin-right: 4px;
+  text-transform: uppercase;
+}
+
+.item-cases {
+  font-family: $FONT-SEMIBOLD;
```

Comment:
@DiscoElevator, should we write fallback like `sans-serif` when we use font constants?

Reply:
No.
I think we need to add some fallback to constants in the future (not in the scope of this PR)